### PR TITLE
[panw_cortex_xdr] Bugfix on rename processor

### PR DESCRIPTION
**Pull request by kcreddy**
## What does this PR do?
Handling multiple values into ECS fields by fixing rename processors with conditionals.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues
Closes https://github.com/elastic/integrations/issues/3957

### CODE REVIEW

File: packages/panw_cortex_xdr/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -49,6 +49,11 @@ processors:
       formats:
         - UNIX_MS
       if: ctx.panw_cortex?.xdr?.events?.event_timestamp != null
+  - date:
+      field: panw_cortex.xdr.events.agent_host_boot_time
+      formats:
+        - UNIX_MS
+      if: ctx.panw_cortex?.xdr?.events?.agent_host_boot_time != null
```
Comment (r00tu53r) [inline packages/panw_cortex_xdr/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml:56]: I see `panw_cortex.xdr.events.event_timestamp` and `panw_cortex.xdr.events.agent_host_boot_time` can both be present. With this change `panw_cortex.xdr.events.agent_host_boot_time` will override `panw_cortex.xdr.events.event_timestamp`. Is that intended ? If so a comment would help. If not -

```suggestion
      if: ctx.panw_cortex?.xdr.events?.event_timestamp == null && ctx.panw_cortex?.xdr?.events?.agent_host_boot_time != null
```

**Comment by LaZyDK** [comment]
Is there any progress on getting this fixed? :)

File: packages/panw_cortex_xdr/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (kcreddy) [reply packages/panw_cortex_xdr/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml:57]: Added `target_field` so it wont update `@timestamp`

File: packages/panw_cortex_xdr/data_stream/alerts/fields/fields.yml [hunk 2]
```diff
@@ -77,6 +81,9 @@
       type: keyword
       description: |
         The ID of the alert.
+    - name: alert_type
+      type: keyword
+      description: "The type of the alert.   \n"
```
Comment (efd6) [inline packages/panw_cortex_xdr/data_stream/alerts/fields/fields.yml:86]:
```suggestion
      description: The type of the alert.
```

**Review by efd6 — COMMENTED** [review]
This will need changes in manifest.yml and changelog.yml.

File: packages/panw_cortex_xdr/data_stream/alerts/fields/fields.yml (hunk 2 above)
Comment (kcreddy) [reply packages/panw_cortex_xdr/data_stream/alerts/fields/fields.yml:86]: Added changelog

**Review by P1llus — APPROVED** [review]

**Pull request merged**